### PR TITLE
Merginbranch

### PR DESCRIPTION
**Pull request by kasarama**

### CODE REVIEW

**Review by Jean-Poul — COMMENTED** [review]
- lombok dependency used for @Data?
- Multipartfile (is it part of filestorageservice?). Not sure I completely understand the filestorage file so if you can explain it in school I would be happy :)
- CustomerError has a super() call in constructor but it doesn't extend anything so you are calling Objects constructor?
- In EmailServiceImpl line 27-29 you catch an exception to then set the country to "DK" how come?

File: assignment-2/src/main/java/dk/jplm/si/assignment2/service/EmailServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    @Autowired
+    GenderServiceImpl genderService;
+
+    @Override
+    public String sendEmail(String email) throws Exception {
+        throw new Exception("Not implemented yet");
+    }
+
+    private List<Guest> setCountries(List<Guest> guests) {
+        for (Guest g : guests
```
Comment (TheRiisager) [inline assignment-2/src/main/java/dk/jplm/si/assignment2/service/EmailServiceImpl.java:23]: This formatting seems a bit strange to me

**Review by TheRiisager — COMMENTED** [review]
Throws the following error when testing: `org.springframework.web.method.annotation.MethodArgumentConversionNotSupportedException: Failed to convert value of type 'org.springframework.web.multipart.support.StandardMultipartHttpServletRequest$StandardMultipartFile' to required type 'java.lang.String'`

additionally, i cannot get it to run without adding the following to pom.xml:
```xml
        <dependency>
            <groupId>javax.xml.rpc</groupId>
            <artifactId>javax.xml.rpc-api</artifactId>
            <version>1.1.1</version>
        </dependency>
        <dependency>
            <groupId>org.apache.axis</groupId>
            <artifactId>axis</artifactId>
            <version>1.4</version>
        </dependency>```

File: assignment-2/src/main/java/dk/jplm/si/assignment2/service/EmailServiceImpl.java (hunk 1 above)
Comment (kasarama) [reply assignment-2/src/main/java/dk/jplm/si/assignment2/service/EmailServiceImpl.java:23]: I am not sire what you mean. Can you explain that to me pls?

**Pull request merged**